jmx-metrics/build.gradle

```diff
@@ -38,25 +39,36 @@ def deps = [
     testcontainers : "org.testcontainers:testcontainers:1.15.1",
 ]
 
+repositories {
+    mavenCentral()
+    maven {
+        setUrl("http://www.terracotta.org/download/reflector/releases")
+        content {
+            includeGroup("org.terracotta")
+        }
+    }
+    mavenLocal()
+}
+
 dependencies {
     implementation "io.grpc:grpc-netty-shaded:${versions.grpc}",
             "org.codehaus.groovy:groovy-jmx:${versions.groovy}",
             "org.codehaus.groovy:groovy:${versions.groovy}",
             "io.prometheus:simpleclient:${versions.prometheus}",
             "io.prometheus:simpleclient_httpserver:${versions.prometheus}",
-            libraries.otelApi,
-            libraries.otelApiMetrics,
-            libraries.otelAutoconfig,
-            libraries.otelExporterLogging,
-            libraries.otelExporterOtlp,
-            libraries.otelExporterPrometheus,
-            libraries.otelSdk,
-            libraries.otelSdkMetrics,
-            libraries.otelSdkTesting,
+            "io.opentelemetry:opentelemetry-api",
+            "io.opentelemetry:opentelemetry-api-metrics",
+            "io.opentelemetry:opentelemetry-sdk",
+            "io.opentelemetry:opentelemetry-sdk-metrics",
+            "io.opentelemetry:opentelemetry-sdk-extension-autoconfigure",
+            "io.opentelemetry:opentelemetry-sdk-testing",
+            "io.opentelemetry:opentelemetry-exporter-logging",
+            "io.opentelemetry:opentelemetry-exporter-otlp-metrics",
+            "io.opentelemetry:opentelemetry-exporter-prometheus",
             deps.slf4j,
             dependencies.create(group: 'org.slf4j', name: 'slf4j-simple', version: versions.slf4j)
 
-    runtime "org.terracotta:jmxremote_optional-tc:1.0.5"
+    runtime "org.terracotta:jmxremote_optional-tc:1.0.8"
```

Comment:
New repo only had 1.0.6+

Reply:
On a clean build I see 

```
Could not determine the dependencies of task ':jmx-metrics:shadowJar'.
> Could not resolve all dependencies for configuration ':jmx-metrics:runtimeClasspath'.
   > Could not resolve org.terracotta:jmxremote_optional-tc:1.0.8.
     Required by:
         project :jmx-metrics
      > Could not resolve org.terracotta:jmxremote_optional-tc:1.0.8.
         > Could not parse POM http://www.terracotta.org/download/reflector/releases/org/terracotta/jmxremote_optional-tc/1.0.8/jmxremote_optional-tc-1.0.8.pom
            > Could not find org.terracotta.forge:forge-parent:4.12.
              Searched in the following locations:
                - https://repo.maven.apache.org/maven2/org/terracotta/forge/forge-parent/4.12/forge-parent-4.12.pom
```

Reply:
Oops accidentally reverted a change for this